holidays/countries/eritrea.py

```diff
@@ -0,0 +1,181 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import JAN, FEB, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC
+from holidays.calendars.julian import JULIAN_CALENDAR
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Eritrea(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """Eritrea holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Eritrea>
+        * <https://web.archive.org/web/20230313234629/https://explore-eritrea.com/working-hours-and-holidays/>
+        * <https://web.archive.org/web/20110903130335/http://www.eritrea.be/old/eritrea-religions.htm>
+        * <https://web.archive.org/web/20250807083700/https://www.timeanddate.com/calendar/?year=2025&country=168>
+        * <https://www.mintageworld.com/media/detail/11411-fenkil-day-in-eritrea/>
+    """
+
+    country = "ER"
+    default_language = "en_ER"
```

Comment:
Honestly, I'm skeptical about the existence of "Eritrean English". If we don't have a reliable source of Tigrinya names, it's better not to do any localization, just use common English names.

Reply:
It's mostly Meskel and Timket vs Finding of True Cross and Epiphany in original sources for example

Reply:
As I understand, it's just original names written in English.